.github/workflows/pr-analyzer-threestage.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   claude-code-pr-review:
     runs-on: ubuntu-latest
     timeout-minutes: 60
-    if: "!startsWith(github.base_ref, 'release-') && !contains(github.event.pull_request.labels.*.name, 'O-No-AI-Review') && github.event.pull_request.merged == false"
+    if: "!startsWith(github.base_ref, 'release-') && !contains(github.event.pull_request.labels.*.name, 'O-No-AI-Review') && github.event.pull_request.merged == false && github.event.pull_request.draft == false"
```

Comment:
Nit: prefer `!github.event.pull_request.draft` to `github.event.pull_request.draft == false`, but I do see we're already using `== false` in this condition anyway so it's no big deal.